specification/storage/data-plane/Microsoft.BlobStorage/preview/2019-12-12/blob.json

```diff
@@ -5837,10 +5839,10 @@
                 "type": "string",
                 "description": "This header uniquely identifies the request that was made and can be used for troubleshooting the request."
               },
-              "x-ms-version": {
-                "x-ms-client-name": "Version",
+              "x-ms-version-id": {
```

Comment:
I don't think we want to delete x-ms-version header from the response. It's not the same thing as x-ms-version-id. x-ms-version returns the service version that Client side is trying to hit.

BTW did we get frontend confirmation the adding x-ms-version-id for abort copy?

Reply:
Oops, it's a mistake. I intended to undo the `x-ms-version-id`.